code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -163,20 +166,27 @@
 	r_hand = /obj/item/melee/transforming/energy/sword/saber
 	mask = /obj/item/clothing/mask/chameleon/gps
 	suit = /obj/item/clothing/suit/armor/vest
-	backpack_contents = list(/obj/item/modular_computer/tablet/preset/syndicate=1)
+	backpack_contents = list(
+		/obj/item/modular_computer/tablet/preset/syndicate=1,
+		/obj/item/storage/box/syndie
```

Comment:
These are ordered differently from the rest
```suggestion
		/obj/item/storage/box/syndie,
		/obj/item/modular_computer/tablet/preset/syndicate=1
```

Reply:
good eye